src/Adapters/Files/S3Adapter.js

```diff
@@ -20,7 +20,9 @@ export class S3Adapter extends FilesAdapter {
       directAccess = false } = {}
   ) {
     super();
-
+    if (!accessKey || !secretKey) {
```

Comment:
That breaks backwards compatibility though :( If you want to have it accept both, that would be cool, but also a lot more effort.